assets/airports/cyhz.json

```diff
@@ -0,0 +1,2633 @@
+{
+    "radio": {
+        "twr": "Halifax Tower",
+        "app": "Halifax Terminal Control",
+        "dep": "Halifax Departure"
+    },
+    "icao": "CYHZ",
+    "iata": "YHZ",
+    "magnetic_north": -18,
+    "ctr_radius": 80,
+    "ctr_ceiling": 18000,
+    "initial_alt": 5000,
+    "position": ["N44d52m52.73", "W063d30m33.62", "477.0ft"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N44d52m47.00", "W063d30m37.00"],
+    "has_terrain": true,
+    "wind": {
+        "angle": 202,
+        "speed": 10
+    },
+    "arrivalRunway": "23",
+    "departureRunway": "23",
+    "airspace": [
```

Comment:
Comment this and add the perimeter like in f/108

Reply:
Huh, that didn't push either?